config/jenkins-configuration.yaml

```diff
@@ -2,6 +2,8 @@
 jenkins:
   systemMessage: "Welcome to Jenkins Essentials!"
   numExecutors: 0
+
+unclassified:
```

Comment:
I haven't been able to follow all the pull request discussions about this on jenkinsci/configuration-as-code-plugin.

I haven't seen this documented anywhere, so perhaps a comment in this file to explain when this might become "unclassfieid?" Everything else in the YAML file is fairly self explanatory or understandable to me, but the special distinction of this tree of configuration is a bit wacky to me :slightly_frowning_face:

Reply:
@rtyler IIUC, that new root got introduced as a default for plugins that do not specify an explicit categorization they should be put in.

Coming from the core `@Symbol` value:
https://github.com/jenkinsci/jenkins/blob/79ec930a8ef4f12d2f2574c50c2f42272e5c75fb/core/src/main/java/jenkins/model/GlobalConfigurationCategory.java#L62

Can you possibly give a bit more precisions @ndeloof @ewelinawilkosz? Thanks!

Reply:
as I implemented yaml generation from a live instance, I wasn't able to distinguish legitimate Jenkins instance attribute vs all the orphaned descriptors we used to attach to it. On the other hand, we already had moved all well categorized descriptors to dedicated root elements (`tools`, `security`...). So I remove the special handling of those uncategorized descriptors to root.